readme.md

```diff
@@ -82,6 +82,10 @@ mixpanel.people.clear_charges('billybob');
 // delete a user
 mixpanel.people.delete_user('billybob');
 
+// delete a user in Mixpanel Engage without altering $last_seen
+// - pass option `$ignore_time: true` to prevent the $last_seen property from being updated
+mixpanel.people.delete_user('billybob', {$ignore_time: true});
```

Comment:
Interesting. I don't feel great about relying on an undocumented implementation detail (that a 'delete' action leaves around the association of distinct ID -> last seen); but on the other hand it seems to be the only way to preserve last seen if reimporting old data, so I'll merge this for the time being, and hopefully we can update the ingestion code in the future to behave a little more sanely for the underlying issue (i.e. don't update last seen as part of a delete action).